src/pages/AdminDashboard/AdminDashboard.jsx

```diff
@@ -1,16 +1,57 @@
 import React from 'react';
+import { ChakraProvider, Button } from '@chakra-ui/react';
+
 import Map from '../../components/Map/Map';
 import BoxApproval from '../../components/BoxApproval/BoxApproval';
+import NavBar from '../../components/NavBar/NavBar';
 import './AdminDashboard.css';
 
 function AdminDashboard() {
+  const sideBarComponent = document.getElementsByClassName('side-bar')[0];
+  const mapComponent = document.getElementsByClassName('map')[0];
+  const reviewSubmissionComponent = document.getElementsByClassName('review-submission-button')[0];
+
+  const setReviewSubmission = () => {
+    sideBarComponent.style.display = 'block';
+    mapComponent.style.width = '80vw';
+    reviewSubmissionComponent.style.display = 'none';
+  };
+
+  const setCloseButton = () => {
+    sideBarComponent.style.display = 'none';
+    mapComponent.style.width = '100vw';
+    reviewSubmissionComponent.style.display = 'block';
+  };
+
   return (
     <div className="admin-dashboard-container">
-      <div className="side-bar">
-        <BoxApproval />
+      <div className="admin-nav-bar">
+        <NavBar />
       </div>
-      <div className="maps">
-        <Map />
+      <div className="side-bar-and-map-container">
+        <div className="side-bar">
+          <BoxApproval />
+          <Button
+            variant="link"
+            colorScheme="white"
+            className="close-button"
+            onClick={() => setCloseButton()}
+          >
+            Close
+          </Button>
+        </div>
+        <div className="map">
+          <Map />
+        </div>
+        <ChakraProvider>
```

Comment:
Could you move the ChakraProvider to wrap around the entire div in this file rather than just the Button component (in the case we need to use more Chakra components later on)?